src/navigator/DrawerNavigator.tsx

```diff
@@ -224,9 +226,10 @@ export default function DrawerNavigator({ route }: Props) {
 
   const shouldShowNftGallery = getFeatureGate(StatsigFeatureGates.SHOW_IN_APP_NFT_GALLERY)
 
-  const shouldShowKeylessBackup = getFeatureGate(
-    StatsigFeatureGates.SHOW_CLOUD_ACCOUNT_BACKUP_SETUP
-  )
+  const cloudBackupGate = getFeatureGate(StatsigFeatureGates.SHOW_CLOUD_ACCOUNT_BACKUP_SETUP)
+  const anyBackupCompleted = backupCompleted || cloudBackupCompleted
+  const shouldShowWalletSecurity = !anyBackupCompleted && cloudBackupGate
+  const shouldShowRecoveryPhrase = !anyBackupCompleted && !cloudBackupGate
```

Comment:
I think we should avoid "shouldShow" prefixes

Reply:
fixed